src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxTreeFactoryService.vb

```diff
@@ -40,19 +40,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return _parseOptionsWithLatestLanguageVersion
             End Function
 
-            Public Overrides Function ParseSyntaxTree(filePath As String, options As ParseOptions, text As SourceText, treeDiagnosticReportingOptionsOpt As ImmutableDictionary(Of String, ReportDiagnostic), cancellationToken As CancellationToken) As SyntaxTree
+            Public Overrides Function ParseSyntaxTree(filePath As String, options As ParseOptions, text As SourceText, analyzerConfigOptionsResult As AnalyzerConfigOptionsResult?, cancellationToken As CancellationToken) As SyntaxTree
                 If options Is Nothing Then
                     options = GetDefaultParseOptions()
                 End If
 
-                Return SyntaxFactory.ParseSyntaxTree(text, options, filePath, treeDiagnosticReportingOptionsOpt, cancellationToken)
+                Return SyntaxFactory.ParseSyntaxTree(text, options, filePath, analyzerConfigOptionsResult?.TreeOptions, cancellationToken)
```

Comment:
Worth adding a comment that this isn't doing the generated flag like the C# one because it's an intentional omission for now?